src/flb_record_accessor.c

```diff
@@ -487,11 +487,30 @@ flb_sds_t flb_ra_translate(struct flb_record_accessor *ra,
                            char *tag, int tag_len,
                            msgpack_object map, struct flb_regex_search *result)
 {
-    int found;
+    int check = FLB_FALSE;
+
+    return flb_ra_translate_check(ra, tag, tag_len, map, result, check);
```

Comment:
How about not using check variable like `return flb_ra_translate_check(ra, tag, tag_len, map, result, FLB_FALSE);` ?

Reply:
fixed